tests/model/model_flattening.cpp

```diff
@@ -657,6 +657,12 @@ TEST(ModelFlattening, unitsUsedByVariableNotInDirectlyImportedComponent)
         "  </encapsulation>\n"
         "</model>\n";
 
+    auto parser = libcellml::Parser::create();
+    auto e_model = parser->parseModel(e);
+    auto validator = libcellml::Validator::create();
+    validator->validateModel(e_model);
+    EXPECT_EQ(size_t(0), validator->errorCount());
```

Comment:
I don't see the purpose of this addition for this test.

Reply:
Done.